### ISSUE TITLE
Feature request: horizontal scrollbar below pinned rows

### ISSUE THREAD

**Issue by egidijusj** [ag-grid/ag-grid#1970]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Consider a case:
- there are more columns that fit into screen
- the columns are not autosized to fit the screen
- having pinned rows on bottom

In such case agGrid renders a horizontal scrollbar between main viewport and pinned rows.

**Expected behavior**
Ideally there would be an option to layout the viewports in a way that horizontal scrollbar remains at the very bottom of the grid, below the pinned rows.

**Minimal reproduction of the problem with instructions**
Plunker: https://plnkr.co/edit/moBppSIQDE7IsMwDLero?p=preview
Relevant part:
![image](https://user-images.githubusercontent.com/5106211/31354503-4cb7a6fc-ad3f-11e7-8bb9-390bb8b822c2.png)

**What is the motivation / use case for changing the behavior?**
It is difficult to override this behaviour and this layout makes custom, more elaborate designs very difficult to implement. And totally subjective, IMO having a scrollbar between rows is very bad UX-wise.

**Please tell us about your environment:**

Applicable to all environments

* **ag-Grid version:** 13.3.1

* **Browser:**
Applicable to all browsers

* **Language:** ES5

**Comment by petyosi** [comment]
Thank you for your suggestion. I have logged it as  AG-975, our dev team will review it.

**Comment by makinggoodsoftware** [comment]
Hi,

Please feel free to track the progress of this issue in our pipeline:

https://www.ag-grid.com/ag-grid-pipeline/

**Comment by egidijusj** [comment]
@makinggoodsoftware - I cannot find AG-975 in the pipeline, what is the resolution on this?

**Comment by jarjee** [comment]
I can't find `AG-975`, but it sounds like it's actually `AG-2072`. May I ask why this feature request is currently parked? Any ETA?

**Comment by CharlesMc** [comment]
Any update on this?

**Comment by reesewesterhoff** [comment]
It looks like this is something they fixed in version 20. I just ran into this problem yesterday and after searching for a while in the documentation I found this example in the CSV export section. https://plnkr.co/edit/?p=preview . There is a pinned bottom row and the scroll bar is beneath the pinned row. To test that it was a version problem, I changed the plunker to version 19 and sure enough, the scroll bar moved from below the bottom pinned row to above it. Hopefully this helps!
